pre_commit/checkers/ruby_version_checker.rb

```diff
@@ -33,7 +33,7 @@ def examine_code
 
   def self.use_for_project?
     val = PreCommitHelper.git_config_val_for_hook(HOOK_KEY)
-    val.empty? || (val == 'false') || @pref_on
+    val.empty? || (val == 'true') || @pref_on
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.